<ui version="4.0" >
 <class>VolumeVisClass</class>
 <widget class="QWidget" name="VolumeVisClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VolumeVis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>4</number>
      </property>
      <property name="margin" >
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnParasagittal" >
        <property name="text" >
         <string>P</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>Resources/Icons/P.ico</normaloff>Resources/Icons/P.ico</iconset>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCoronal" >
        <property name="text" >
         <string>C</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>Resources/Icons/C.ico</normaloff>Resources/Icons/C.ico</iconset>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAxial" >
        <property name="text" >
         <string>A</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>Resources/Icons/A.ico</normaloff>Resources/Icons/A.ico</iconset>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Viewpoint:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="viewCombo" >
        <item>
         <property name="text" >
          <string>Free</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Axial</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Coronal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Sagittal</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sliceCheckBox" >
        <property name="text" >
         <string>Show slice positions</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="dtiFrame" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Display:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dispComboBox" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text" >
             <string>Signal level</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Color</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>FA</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkInterpolate" >
           <property name="text" >
            <string>Show Interpolation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="dispFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
